### PR TITLE
Switch to the honister OE release

### PR DESCRIPTION
**Pull request by lumag**
- switch to delcaring compatibility with `honister`
- convert overrides syntax

### CODE REVIEW

File: recipes-overlayed/busybox/busybox_1.%.bbappend [hunk 1]
```diff
@@ -1,2 +1,2 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
-SRC_URI_append_rpb = " file://enable-setsid-tty.cfg file://enable-install.cfg"
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+SRC_URI:append_rpb = " file://enable-setsid-tty.cfg file://enable-install.cfg"
```
Comment (ndechesne) [inline recipes-overlayed/busybox/busybox_1.%.bbappend:2]: you need :append:rpb here (and perhaps in other places too)

Comment (lumag) [reply recipes-overlayed/busybox/busybox_1.%.bbappend:2]: True

File: conf/distro/include/rpb.inc [hunk 2]
```diff
@@ -30,28 +30,28 @@ def get_multilib_handler(d):
...
 
 # Avoid to duplicate the rootfs tarball by generating both tar.gz/tar.xz
-IMAGE_FSTYPES_remove = "tar.gz"
+IMAGE_FSTYPES:remove = "tar.gz"
 
 # By default, meta-ti and TI SDK ship tar.xz rootfs tarball for SD card deployment
 # fastboot is preferred for deployment in automation
-IMAGE_FSTYPES_append_omap-a15 = " ext4.gz"
-IMAGE_FSTYPES_append_beaglebone = " ext4.gz"
+IMAGE_FSTYPES:append_omap-a15 = " ext4.gz"
```
Comment (ndechesne) [inline conf/distro/include/rpb.inc:53]: miss!

Comment (lumag) [reply conf/distro/include/rpb.inc]: Fixed these ones and few more

**Pull request merged**